pkg/profilecreator/profilecreator_test.go

```diff
@@ -140,3 +140,54 @@ var _ = Describe("PerformanceProfileCreator: Consuming GHW Snapshot from Must Ga
 
 	})
 })
+
+var _ = Describe("PerformanceProfileCreator: Populating Reserved and Isolated CPUs in Performance Profile", func() {
+	var mustGatherDirPath, mustGatherDirAbsolutePath string
+	var node *v1.Node
+	var handle *GHWHandler
+	var splitReservedCPUsAcrossNUMA bool
+	var reservedCPUCount int
+	var err error
+
+	BeforeEach(func() {
+		node = newTestNode("cnfd1-worker-0.fci1.kni.lab.eng.bos.redhat.com")
+		mustGatherDirPath = "../../testdata/must-gather/must-gather.local.directory"
+	})
+	Context("Check if reserved and isolated CPUs are properly populated in the performance profile", func() {
+		It("Ensure reserved CPUs populated are correctly when splitReservedCPUsAcrossNUMA is disabled", func() {
+			reservedCPUCount = 20
```

Comment:
is this a special number of just a random number? (could have been 19, 21, 30...?) If it's just a random number please add a small comment like
```
// random number, no special meaning
```

Reply:
Sure, makes sense, will do.